packit/cli/scan_in_osh.py

```diff
@@ -0,0 +1,71 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+import os
+
+import click
+
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception, get_packit_api, iterate_packages
+from packit.config import (
+    get_context_settings,
+    pass_config,
+)
+from packit.constants import (
+    PACKAGE_LONG_OPTION,
+    PACKAGE_OPTION_HELP,
+    PACKAGE_SHORT_OPTION,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@click.command("scan-in-osh", context_settings=get_context_settings())
+@pass_config
+@cover_packit_exception
+@iterate_packages
+@click.option(
+    PACKAGE_SHORT_OPTION,
+    PACKAGE_LONG_OPTION,
+    multiple=True,
+    help=PACKAGE_OPTION_HELP.format(action="build"),
+)
+@click.option(
+    "--target",
+    help="Chroot to build in. (defaults to 'fedora-rawhide-x86_64')",
+    default="fedora-rawhide-x86_64",
+)
+@click.option(
+    "--base-srpm", help="Base SRPM to perform a differential build", default=None,
+)
+@click.option(
+    "--comment",
+    help="Comment for the build",
+    default="Submitted through PackIt.",
```

Comment:
```suggestion
    default="Submitted through Packit.",
```

We usually don't spell it like this even though it's part of the meaning..;)